LambdaEngine/Source/Rendering/EntityMaskManager.cpp

```diff
@@ -18,6 +18,12 @@ bool LambdaEngine::EntityMaskManager::Init()
 	return true;
 }
 
+void LambdaEngine::EntityMaskManager::RemoveAllExtensionsFromEntity(Entity entity)
+{
+	if(s_EntityToExtensionGroupEntryMap.contains(entity))
+		s_EntityToExtensionGroupEntryMap.erase(entity);
```

Comment:
Just `erase(entity)` is fine, it can handle misses.

Reply:
👌